app/src/main/java/com/android/sample/ui/activity/EditActivity.kt

```diff
@@ -234,76 +239,72 @@ fun EditActivityScreen(
-                        )
-                        Button(
-                            onClick = { attendees = attendees.filter { it != attendees[index] } },
-                            modifier =
-                                Modifier.width(40.dp).height(40.dp).testTag("removeAttendeeButton"),
-                        ) {
-                          Icon(
-                              Icons.Filled.PersonRemove,
-                              contentDescription = "remove attendee",
-                          )
-                        }
-                      }
+          Button(
+              onClick = { showDialog = true },
+              modifier =
+                  Modifier.width(300.dp)
+                      .height(40.dp)
+                      .testTag("addAttendeeButton")
+                      .align(Alignment.CenterHorizontally),
+          ) {
+            Row(
+                horizontalArrangement = Arrangement.spacedBy(8.dp, Alignment.CenterHorizontally),
+                verticalAlignment = Alignment.CenterVertically,
+            ) {
+              Icon(
+                  Icons.Filled.Add,
+                  contentDescription = "add a new attendee",
+              )
+              Text("Add Attendee")
+            }
+          }
+          if (attendees.isNotEmpty()) {
+            LazyRow(
+                modifier = Modifier.fillMaxHeight().height(85.dp).padding(8.dp),
+            ) {
+              items(attendees.size) { index ->
+                Card(
+                    modifier =
+                        Modifier.padding(8.dp)
+                            .background(Color(0xFFFFFFFF))
+                            .testTag("attendeeRow${index}"),
+                ) {
+                  Row {
+                    Column(modifier = Modifier.padding(8.dp)) {
+                      Text(
+                          text = "${attendees[index].name} ${attendees[index].surname}",
+                          modifier = Modifier.testTag("attendeeName${index}"),
+                          style = TextStyle(fontWeight = FontWeight.Bold, fontSize = 12.sp),
+                      )
+                      Text(
+                          text = "Age: ${attendees[index].age}",
+                          modifier = Modifier.testTag("attendeeAge${index}"),
+                          style = TextStyle(fontWeight = FontWeight.Bold, fontSize = 12.sp),
+                      )
+                    }
+                    IconButton(
+                        onClick = { attendees = attendees.filter { it != attendees[index] } },
+                        modifier =
+                            Modifier.width(40.dp).height(40.dp).testTag("removeAttendeeButton"),
+                    ) {
+                      Icon(
+                          Icons.Filled.PersonRemove,
+                          contentDescription = "remove attendee",
+                      )
                     }
```

Comment:
The new implementation reduces redundancy by merging similar modifiers and removing duplicated code lines, especially around button and text implementations.